src/games/strategy/engine/framework/GameRunner.java

```diff
@@ -50,14 +52,18 @@
  */
 public class GameRunner {
 
+  public enum GameMode { SWING_CLIENT, HEADLESS_BOT }
```

Comment:
2 other benefits:
- type safe, do not suffer from incorrect param ordering problem, ie: "doSomething(boolean isHeadless, boolean isSomeOtherOption)"
- if necessary, allows for a third option rather than just a binary